specification/wse/src/main/scripts/org/kaazing/specification/wse/browsers/client.reconnect.downstream/request.rpt

```diff
@@ -7,14 +7,14 @@ write method "POST"
 write version "HTTP/1.1"
 write header host
 write header content-length
-write header "X-WebSocket-Version" "wseb-1.1"
+write header "X-WebSocket-Version" "wseb-1.0"
 write header "X-Sequence-No" ${wse:asString(sequence)}
 write close
 
 read status "201" /.+/
 read version "HTTP/1.1"
-read header "X-WebSocket-Version" "wseb-1.1"
-read header "Content-Type" "text/plain;charset=utf-8"
+read header "X-WebSocket-Version" "wseb-1.0"
+read header "Content-Type" /text\/plain;charset=(?i)utf-8/
```

Comment:
It's just to make the charset value case insensitive. I have seen cases where server would return "text/plain;charset=UTF-8". Semantically it is equivalent to "text/plain;charset=utf-8". It may not be robust for all combinations but we can fix that as we come across those scenarios.